src/app/pages/trip-summary/trip-summary.component.scss

```diff
@@ -0,0 +1,88 @@
+@import url('https://fonts.googleapis.com/css2?family=Yeseva+One&display=swap');
```

Comment:
No need to import fonts here, there is a link in the index.html that loads all the needed fonts.